lib/Dialect/SV/Transforms/HWMemSimImpl.cpp

```diff
@@ -33,7 +35,27 @@ struct FirMemory {
   size_t readLatency;
   size_t writeLatency;
   size_t readUnderWrite;
+  size_t writeUnderWrite;
+  llvm::EquivalenceClasses<unsigned> writeSets;
 };
+
+class SymbolDigger {
+  DenseMap<Attribute, SmallVector<Operation *, 1>> internal;
+
+public:
+  SymbolDigger(ModuleOp module) {
+    module.walk([&](Operation *op) {
+      TypeSwitch<Operation *, void>(op)
+          .Case<hw::InstanceOp>([&](hw::InstanceOp op) {
+            internal[op.moduleNameAttr()].push_back(op);
```

Comment:
If all you are doing is getting the instances using a module, that should work with the existing symbol table stuff.  The Use discover will recurse and pick up instances using module symbols.

Reply:
This is now gone after the reworking this to add clock information during `LowerToHW`.